config/sitemap.rb

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
Can this entire thing be moved to a Rake task? That will give us a lot of benefits like:

 - RSolr already available to us and configured for the specific environment
 - Use of rails_config gem for env specific configuration options